src/System.Runtime/tests/TestModule/System.Reflection.TestModule.ilproj

```diff
@@ -1,13 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+<Project Sdk="Microsoft.NET.Sdk.IL">
   <PropertyGroup>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
     <ProjectGuid>{3B7489C4-65DB-4E69-BE01-F6234133400C}</ProjectGuid>
     <Configurations>Debug;Release</Configurations>
+    <IsTestProject>false</IsTestProject>
```

Comment:
Why was setting that required?

Reply:
Otherwise it will get the Open key, which is a full key pair, and ILAsm will fail because it can't fully sign assemblies on .NETCore.  Previously the assembly used the Microsoft key because at the time it was built it wasn't covered by our test project convention (eg: projectname.endswith(.tests)

Reply:
Can you instead just override the keytype? IsTestProject is used for more than just the key.

Reply:
It's set unconditionally in a targets: https://github.com/dotnet/corefx/blob/c8e44c3d027d0539e7f9091336cc48e59e7d03e8/Directory.Build.targets#L14

I could change that, but I don't think its necessarily wrong to say that this isn't a test project.  It doesn't contain any tests.

Reply:
One change that could help there would be to move that to directory.build.props after IsTestProject is set, but as I mentioned it didn't feel "wrong" to me to set IsTestProject to false.  Let me know what you think.

Reply:
We have the notion of a `TestProject` and a `TestSupportProject` which also counts as a TestProject. This project is a perfect example for a `TestSupportProject`. It might not use any of the features that we offer such projects but I think it should be marked correctly.

https://github.com/dotnet/arcade/blob/18cf504e68343301e5b4d9758cd690bdc1b00015/src/Microsoft.DotNet.CoreFxTesting/build/TestProject.props#L24-L25

Reply:
That still wouldn't solve the signing problem...

Reply:
I didn't propose a solution 😅 just wanted to let you know why I think the project should keep the `IsTestProject` flag.

Reply:
I see.  I will try my suggested alternative change and write a test target that dumps final value of key for all projects to do a diff of before and after to ensure it doesn't change other projects.

Reply:
Thanks :)

Reply:
So it looks like my suggestion isn't sufficient because all libraries define their StrongNameId after importing root directory.build.props.  https://github.com/dotnet/corefx/blob/4d0d12ca15f70f1250a648fd4b1294a49e344ef6/src/System.Runtime/Directory.Build.props#L3-L6

So moving the property from targets to props will undo its effect.  Refactoring all those library-specific-props will cause numerous cascading problems.

I'll add a test-specific property to indicate the key.